backend/typescript/rest/contentRoutes.ts

```diff
@@ -0,0 +1,48 @@
+import { Router } from "express";
+import ContentService from "../services/implementations/contentService";
+import IContentService from "../services/interfaces/contentService";
+import getErrorMessage from "../utilities/errorMessageUtil";
+import { sendResponseByMimeType } from "../utilities/responseUtil";
+import contentDtoValidator from "../middlewares/validators/contentValidators";
+
+const contentRouter: Router = Router();
+const contentService: IContentService = new ContentService();
+
+contentRouter.post("/", async (req, res) => {
```

Comment:
wont really matter but maybe add contentDtoValidator in post as well

Reply:
won't we not be using this, since the FE to "post" content isn't going to be exposed? We'll just insert this in Pgadmin. But added 👍🏻